modules/organization/main.tf

```diff
@@ -16,13 +16,17 @@
 
 locals {
   iam_additive_pairs = flatten([
-    for member, roles in var.iam_additive : [
-      for role in roles :
-      { role = role, member = member }
+    for role, members in var.iam_additive : [
+      for member in members : { role = role, member = member }
+    ]
+  ])
+  iam_additive_member_pairs = flatten([
```

Comment:
isn't [transpose()](https://www.terraform.io/docs/configuration/functions/transpose.html) what you want?